xbee/__init__.py

```diff
@@ -5,10 +5,6 @@
 """
 
 __title__ = 'xbee'
-__version__ = '2.2.5'
+__version__ = '2.3.0'
```

Comment:
I wasn't certain, but since your comment, that this is a non BC change I wanted to bump a minor version in here.